core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -326,82 +318,10 @@ public Options checkConsistency() {
     public Options fetch(ColumnType... colsToFetch) {
       Preconditions.checkArgument(colsToFetch.length > 0);
 
-      for (ColumnType colToFetch : colsToFetch) {
-
-        fetchedCols.add(colToFetch);
```

Comment:
Will still need to add things to the set of fetchedColumns

Reply:
Yep that would indeed be a problem and is what caused the test failures, should be fixed now.